### PR TITLE
Ignore IPAM requests for space until we have a ring

### LINKED ISSUE #2275: Unable to claim address immediately after ring is formed
````
host1$ weave reset
host2$ weave reset
host1$ weave launch $HOST2
host2$ weave launch $HOST1
host2$ docker $(weave config) run --rm -e WEAVE_CIDR=10.32.1.1/12 -ti alpine true
docker: Error response from daemon: Unable to claim: address 10.32.1.1/12 is owned by other peer 0e:2a:9b:8e:23:fd (host1)
````

It appears that the unicast request for space arrives before the broadcast of the initial ring.

### PR DESCRIPTION
**Pull request by bboreham**
Fixes #2275

In the case that we do receive a ring very soon, the consequence of this change is that the requestor doesn't hear back so retries within 5 seconds.

I also moved the debug message so we still see it on requests that are ignored.

### CODE REVIEW

File: test/320_claim_3_test.sh [hunk 1]
```diff
@@ -13,12 +14,11 @@ delete_persistence() {
 
 start_suite "Claiming addresses"
 
-weave_on $HOST1 launch-router --ipalloc-range $UNIVERSE
+weave_on $HOST1 launch-router --ipalloc-range $UNIVERSE $HOST2
```
Comment (brb) [inline test/320_claim_3_test.sh]: Why is it needed?

Comment (bboreham) [inline test/320_claim_3_test.sh]: without this change, host1 can form a ring without talking to host2, but I want to test what happens when it has to wait for consensus.

File: test/320_claim_3_test.sh [hunk 2]
```diff
@@ -13,12 +14,11 @@ delete_persistence() {
 
 start_suite "Claiming addresses"
 
-weave_on $HOST1 launch-router
-weave_on $HOST2 launch-router $HOST1
+weave_on $HOST1 launch-router --ipalloc-range $UNIVERSE $HOST2
```
Comment (brb) [inline test/320_claim_3_test.sh]: Why passing `$HOST2` is needed?

**Comment by brb** [comment]
Have you considered calling `alloc.tryPendingOps()` after establishing the ring? It could make re-try to happen sooner.

**Comment by brb** [comment]
LGTM.

File: test/320_claim_3_test.sh (hunk 2 above)
Comment (bboreham) [inline test/320_claim_3_test.sh]: Without this change, host1 can form a ring without talking to host2, but I want to test what happens when it has to wait for consensus.  I could equally have told it a peer count of 2.

**Pull request merged**